server/src/main.rs

```diff
@@ -21,6 +21,7 @@ use rustls::crypto::aws_lc_rs;
 use tracing::error;
```

Comment:
In this crate there are a few integration tests () where we probably should start the `ClockUpkeep` as well, if this is not already happening.